tapgarden/planter_test.go

```diff
@@ -1646,6 +1659,25 @@ func testFundBeforeFinalize(t *mintingTestHarness) {
 	t.assertPendingBatchExists(numSeedlings)
 	t.assertSeedlingsExist(seedlings, nil)
 
+	observedGroupCount := 0
+	fundedSeedlings, err := t.planter.ListBatches(
+		tapgarden.ListBatchesParams{
+			Verbose: true,
+		},
+	)
+	t.Logf("%v", spew.Sdump(fundedSeedlings[0]))
```

Comment:
nit: stray debug log. Maybe add some additional context if we want to keep it in for debugging purposes?